src/azure-cli/azure/cli/command_modules/appservice/tests/latest/test_webapp_commands.py

```diff
@@ -107,6 +107,9 @@ def test_webapp_e2e(self, resource_group):
             JMESPathCheck('httpsOnly', False),
         ])
 
+    def test_webapp_runtimes(self):
+        self.cmd('webapp list-runtimes')
```

Comment:
Should we check that we get status 200 ?

Reply:
This test case is so simple. Maybe you can add some assertions.

Reply:
I just want to test the command will succeed. If the command fails the test will failed, too.

Reply:
For example, check the returned values, check their format, check not empty output